### ISSUE TITLE
Feature request: Open access in journal article metadata

### ISSUE THREAD

**Issue by alainna** [CrossRef/rest-api-doc#448]
As an open access repository, it would be useful to be able to know from the DOI metadata whether the article is open access / in an open access journal.

**Comment by bnewbold** [comment]
Hi @alainna, I have similar interests.

The API does return license information if the publisher has deposited it. If there is a creative commons (or other OA) license on the version of record ("vor"), that's a pretty strong signal that the work is open access, and licenses on other versions can hint that there may be a "green OA" copy somewhere (eg, in a repository). The license URLs are not always "normalized" and need some parsing; if you're interested I have some example code for this here (note that this currently discards region-specific variants of CC licenses):

  https://github.com/internetarchive/fatcat/blob/master/python/fatcat_tools/importers/crossref.py#L46

I've found that currently this paper-level metadata needs to be combined with other journal-level sources (like DOAJ, ISSN ROAD, and SHERPA/RoMEO) to get better coverage of "gold" open access journals (which may not have deposited license URLs with crossref).

You might want to look at the unpaywall.org API, which returns specifically OA status (and locations) for works by DOI.

**Comment by anmipac** [comment]
Could it be that there is still a number of publishers not depositing information on the OA status of an article? I found this example:

http://api.crossref.org/works/10.1080/01402382.2019.1583885

Nothing there indicates that the paper is in fact OA (if I did not overlook anything).

While it works with Unpaywall (https://api.unpaywall.org/v2/10.1080/01402382.2019.1583885?email=YOUR_EMAIL), it seems that there is a time lag of some hours between the moment in which a DOI is deposited at CrossRef and the moment OA information are feeded into Unpaywall's API.
